evaluate/metrics/category.go

```diff
@@ -1,30 +1,77 @@
 package metrics
 
+import "fmt"
+
 // AssessmentCategory represents a categorical ranking of a model based on Assessments.
-type AssessmentCategory string
+type AssessmentCategory struct {
+	// Name holds a unique short name of the category.
+	Name string
```

Comment:
Can you put the refactoring to give assessment categories a name and description in its own commit please